ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -45,7 +45,7 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
 
     this.renderComponent = () =>
       render(
-        hbs`<Secrets::Page::Overview @destinations={{this.destinations}} @totalVaultSecrets={{7}} @activatedFeatures={{this.activatedFeatures}} @isAdapterError={{false}} />`,
+        hbs`<Secrets::Page::Overview @destinations={{this.destinations}} @totalVaultSecrets={{7}} @activatedFeatures={{this.activatedFeatures}} @adapterError={{""}} />`,
```

Comment:
```suggestion
        hbs`<Secrets::Page::Overview @destinations={{this.destinations}} @totalVaultSecrets={{7}} @activatedFeatures={{this.activatedFeatures}} @adapterError={{null}} />`,
```

also trivial, but since the `type` of `adapterError` can either be [`AdapterError | null`](https://github.com/hashicorp/vault/pull/25907/files#diff-1bf2cd013f56790f57c50001226cc6540adbfadd677e167e9837df798689fc8aR27), we should pass in the correct type here. that way we know we're testing with accurate input.

someday we can hopefully start using [ember-template-glint](https://github.com/hashicorp/vault/pull/25907/files#diff-1bf2cd013f56790f57c50001226cc6540adbfadd677e167e9837df798689fc8aR27) which will catch these types of things for us 😀

Reply:
Good catch. These types changed from string to Adapter error || null and I missed the updates everywhere.